etn-ccis/blui-react-workflows
# Add reset password base screen

Fixes BLUI-4012.



#### Changes proposed in this Pull Request:
- Add reset password screen base
- Added defaults and cleaned up passthrough props for finish state and success base components (out of scope, but I kind of needed them working to demo this)



#### Screenshots / Screen Recording (if applicable)

![Screen Shot 2023-05-31 at 2 30 59 PM](https://github.com/etn-ccis/blui-react-workflows/assets/13989985/98b9b4b9-554c-47b7-b055-089484355283)
![Screen Shot 2023-05-31 at 2 30 45 PM](https://github.com/etn-ccis/blui-react-workflows/assets/13989985/70194c6d-59ea-430b-b115-54343a65d36c)



#### To Test:

- `yarn start:example2`
- click "Reset Password" button and ensure that it works as expected




#### Any specific feedback you are looking for?

- Ensure that you can pass a custom Successscreen
- Ensure you can customize the pre-built SuccessScreen.
- There is an example of both of these built into the demo already, you'll just need to uncomment one part to get the custom success screen
